rootfs/scheduler/__init__.py

```diff
@@ -1004,12 +1004,7 @@ def _scale_rc(self, namespace, name, desired):
             'version': rc['spec']['selector']['version']
         }
 
-        # Are there any pods running (and verified as ready) available?
-        pods = self._get_pods(namespace, labels=labels).json()['items']
-        current = 0
-        for pod in pods:
-            if self._pod_ready(pod):
-                current += 1
+        current = int(rc['spec']['replicas'])
```

Comment:
Ah right, that's true. However that is still not going to work. We need to figure out if there are *working* pods ready, not how many pods the RC thinks it should have (that's the difference between `spec/replicas` and `status/replicas`)